docs/yellowpaper/sections/01-introduction/index.tex

```diff
@@ -40,37 +40,30 @@ \subsection{Security Goals}
 
 Generally, the HOPR protocol aims to hide the fact that two parties are communicating with each other, as well as the contents of the communication. This information should be hidden from any party external to the network, all intermediaries involved in transferring the data between the communicating parties, and even the communicating parties themselves, if one or both of them desires it.
 
-Specifically, the HOPR protocol aims to build a network with the following features:
+Specifically, the HOPR protocol aims to build a network with the following features (definitions based on \cite{AnoA} and \cite{sphinxpaper}:
 
 \begin{itemize}
-
-    \item \textbf{Sender-receiver unlinkability}
-          An adversary should be unable to distinguish whether $\{S_1\rightarrow R_1, S_2\rightarrow R_2\}$ or $\{S_1\rightarrow R_2, S_2\rightarrow R_1\}$ for any concurrently online honest senders $S_1$, $S_2$ and honest receivers $R_1$, $R_2$ of the adversary’s choice.
     \item \textbf{Sender online unobservability}
-          An adversary should be unable to determine whether or not a specific sender $S$ is communicating with any receiver $R$, for any concurrently online honest sender $S$ of the adversary’s choice.
-          \begin{center}
-              Sender online unobservability $\rightarrow$ sender anonymity
-          \end{center}
-          $\{S_1 \rightarrow R,S_2 /\rightarrow \;or\; S_1 /\rightarrow,S_2 \rightarrow R\}$ for any concurrently online honest senders $S_1$ and $S_2$ and any receiver of the adversary’s choice.
-
-
+        For any sender, $A$, it should be impossible for an adversary to determine whether they are communicating with any receiver, $Z$ (denoted as $A \rightarrow Z$).
+    \item \textbf{Sender anonymity}
+        $\{S_1 \rightarrow R,S_2 /\rightarrow \;or\; S_1 /\rightarrow,S_2 \rightarrow R\}$ for any concurrently online honest senders $S_1$ and $S_2$ and any receiver of the adversary’s choice.
+    \item \textbf{Sender-receiver unlinkability}
+    	For any pair of senders, $A_1 and A_2$, communicating with any pair of receivers, $Z_1 and Z_2$, an adversary must be unable to determine whether $\{A_1\rightarrow Z_1, A_2\rightarrow Z_2\}$ or $\{A_1\rightarrow Z_2, A_2\rightarrow Z_1\}$.
```

Comment:
Not sure if this is equivalent, but sounds like that.

Reply:
That was my concern. But then what was the difference between this and the original set of definitions?